### ISSUE TITLE
make should stop upon error

### ISSUE THREAD

**Issue by mmokrejs** [jts/nanopolish#148]
Hi Jared,
  I have some compile issues but surprisingly make continues to install step (read `make` which should translate to `make compile` even starts **installing** the files to `$(DESTDIR)` system-wide location). I just screwed up my environment because of this as it partially overwrote my hdf5-1.8.18 with a piece of 1.8.15.

  Anyway, this code in Makefile is nasty:

```
cd hdf5-1.8.14; ./configure --enable-threadsafe --prefix=`pwd`/..; make; make install
```

Use `&&` instead. Seems the Makefile shoould use in more places ` || exit 255` to properly exit upon error.

**Comment by jts** [comment]
Fixed in 97987fc

**Comment by mmokrejs** [comment]
I think there are more places in the `Makefile` which deserve something similar. There is a bunch of `cd` commands which could fail. wget could fail as well as tar.
